### DISCUSSION TITLE
Handsome Profile

### DISCUSSION THREAD

**Discussion by azvast** [community/community#18768] [Other Feature Feedback, Questions, & Ideas]
How can I make a handsome GitHub profile so that someone sees my development history more easily?

**Comment by WindingSnow** [comment]
Create a repository with a name the same as your GitHub profile name. The repository's readme will appear on your profile. You can add information about yourself so that others can know about you. For reference-[My Profile](https://github.com/Saarthak-Slathia)

**Reply by blockchainGuru1018** [reply]
but you didn't get the badge yet.
how can I trust your way?

**Reply by aasisodiya** [reply]
> but you didn't get the badge yet. how can I trust your way?

What badge?

**Comment by ckc9759** [comment]
The best way is to explore different GitHub profiles and view their readme profile !!.
I created mine using different sources. 
Markdown language supports HTML, hence you can add images symbols and gifs to make it more attractive.
You can have a look at my GitHub profile for more ideas:  https://github.com/ckc1404

**Comment by Zi-Tao** [comment]
@azvast  These profile templates should be able to help you!
https://github.com/durgeshsamariya/awesome-github-profile-readme-templates
https://github.com/topics/profile-readme
https://github.com/topics/profile-template

**Comment by svintit** [comment]
Feel free to use mine as a starting point if you would like!

**Comment by manasesjesus** [comment]
TLDR; Create a repo named as your username (**azvast**) and add a README. The info in your readme will be displayed on your profile. You can use [this tool](https://rahuldkjain.github.io/gh-profile-readme-generator/) to generate the contents of your readme, and [this one](https://github.com/anuraghazra/github-readme-stats) to add your statistics. Here's GitHub's step-by-step [profile README tutorial](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme).

Another approach is using GitHub Actions to update your readme whenever an event triggers the action. There are actions such as [this](https://github.com/marketplace/actions/github-update-readme), [this](https://github.com/marketplace/actions/update-action-readme), and [many more](https://github.com/marketplace?type=actions&query=readme+). You can add all the info you want to make it handsome.

You can also code your own action using GitHub GraphQL and REST APIs to get the info you want, and update the readme with that info. It can be displayed in SVG and you can use CSS for colors and animations. Take a look at [my profile](https://github.com/manasesjesus) to see this approach.

Finally, take a look at [this repository](https://github.com/abhisheknaiidu/awesome-github-profile-readme) as it has a collection of readme profiles. You may get inspiration by looking at different profiles.

**Comment by cardison** [comment]
HI, @azvast  i saw your profile it seems to be awsome.  but if you like to make your profile even more handsome than that follow these steps
 
1.You can make handsome profile by editing your readme.md  also i have several types of profile that  makes your profile handsome .


2.PROFILES
https://github.com/cardison
https://github.com/MasterBrian99
these are the profiles that i know i hope you like them 

also you can add these badges to make your profile looks  like a professional profile  

3.visit this site to add trophy badge- https://github.com/ryo-ma/github-profile-trophy

HOPE THIS REVIEW HELPS YOU
have a nice day 😊
